src/pk_core.c

```diff
@@ -1302,7 +1302,13 @@ bool varContains(PKVM* vm, Var elem, Var container) {
       return false;
     } break;
 
-    case OBJ_MAP:
+    case OBJ_MAP: {
+      Map* map = (Map*)AS_OBJ(container);
+      return mapGet(map, elem) == VAR_UNDEFINED
+        ? false
+        : true;
```

Comment:
This can be simplified with  `return !IS_UNDEF(mapGet(map, elem));`

Reply:
Neat!